### PR TITLE
Increase visibility of navigation cards and convert entire card to clickable entity

### PR DESCRIPTION
**Pull request by BenWilson2**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/BenWilson2/mlflow/pull/10283?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10283/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10283
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?

Changes to navigation cards:
- Make cards visible when not hovered
- Make the entire card clickable
- Remove the header link behavior
- Simplify the card CSS 

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [X] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [X] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [X] Instructions

#### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [X] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/theme/mlflow/static/css/simple-cards.css [hunk 1]
```diff
@@ -43,79 +48,21 @@
...
+}
+
 .simple-card p {
     color: darkslategrey;
     font-size: 1rem;
     margin: 1rem 2rem; 
     text-align: left; 
 }
-
-.clickable-header {
```
Comment (BenWilson2) [inline docs/theme/mlflow/static/css/simple-cards.css:67]: We no longer need any of this.

File: docs/source/getting-started/index.rst [hunk 2]
```diff
@@ -34,70 +34,70 @@ Interested in navigating directly to the content that you're curious about? Sele
                     <div class="header">
                         Setting up MLflow Servers
                     </div>
+                    <p>
+                        Learn how to start a MLflow Tracking Server and the MLflow UI Server locally
```
Comment (chenmoneygithub) [inline docs/source/getting-started/index.rst:38]:
```suggestion
                        Learn how to start an MLflow Tracking Server and the MLflow UI Server locally
```

File: docs/source/getting-started/index.rst [hunk 3]
```diff
@@ -34,70 +34,70 @@ Interested in navigating directly to the content that you're curious about? Sele
...
-                    Explore the MLflow UI and create your first MLflow experiment with a unique name and identifying tags
-                </p>
             </div>
             <div class="simple-card">
                 <a href="logging-first-model/step4-experiment-search.html" >
                     <div class="header">
                         Search Experiments by tags
                     </div>
+                    <p>
+                        Learn the benefits of relevant identifying tags as you execute a search for experiments containing identifying tag values
```
Comment (chenmoneygithub) [inline docs/source/getting-started/index.rst:68]: this looks a bit complex, shall we just say "Learn how to apply tags to MLflow experiments and search experiments by tags"?

File: docs/source/getting-started/index.rst [hunk 4]
```diff
@@ -34,70 +34,70 @@ Interested in navigating directly to the content that you're curious about? Sele
...
-                    Learn how to start a MLflow Tracking Server and the MLflow UI Server locally
-                </p>
             </div>
             <div class="simple-card">
                 <a href="logging-first-model/step2-mlflow-client.html" >
                     <div class="header">
                         Using the MLflow Client
                     </div>
+                    <p>
+                        Connect to the Tracking Server with the MLflow Client and learn to search for experiments
```
Comment (chenmoneygithub) [inline docs/source/getting-started/index.rst:48]:
```suggestion
                        Connect to the Tracking Server with the MLflow Client and learn how to search for experiments
```

**Review by chenmoneygithub — APPROVED** [review]
Thanks Ben, I checked the rendered page, and it looks great!

One thing I have been thinking about: shall we instead of showing tiles for quickstart 1 and quickstart 2, we just put itself in a tile? I feel it's odd to start reading the quickstart from a certain section: https://output.circle-artifacts.com/output/job/20663c13-c8a4-4b1a-b506-d8d66122e6fc/artifacts/0/docs/build/html/getting-started/index.html#id2

**Comment by BenWilson2** [comment]
> Thanks Ben, I checked the rendered page, and it looks great!
> 
> One thing I have been thinking about: shall we instead of showing tiles for quickstart 1 and quickstart 2, we just put itself in a tile? I feel it's odd to start reading the quickstart from a certain section: https://output.circle-artifacts.com/output/job/20663c13-c8a4-4b1a-b506-d8d66122e6fc/artifacts/0/docs/build/html/getting-started/index.html#id2

We'll be replacing both of these sections with a new "Quickstart" section at the top where the new Tracking Quickstart, the AutoLogging Quickstart, and the Logging Quickstart will go :D we'll do that as part of the post-hackathon work :)

File: docs/source/getting-started/index.rst (hunk 3 above)
Comment (BenWilson2) [inline docs/source/getting-started/index.rst:68]:
```suggestion
                        Learn how to use tags for MLflow experiments and how to leverage them for searching
```

File: docs/source/getting-started/index.rst (hunk 2 above)
Comment (BenWilson2) [reply docs/source/getting-started/index.rst:38]: thanks for the catch :)

**Pull request merged**